organization.go

```diff
@@ -18,9 +18,9 @@ const OrganizationAPIURL = "/v2/organization"
 const OrganizationMemberAPIURL = "/v2/organization/member"
 const OrganizationMembersAPIURL = "/v2/organization/members"
 
-// GetOrganization gets an organization.
-func (c *Client) GetOrganization(ctx context.Context, id string) (*organization.Organization, error) {
-	resp, err := c.doRequest(ctx, "GET", OrganizationAPIURL+"/"+id, nil, nil)
+// GetOrganization gets an organization associated with the token in use by the client.
+func (c *Client) GetOrganization(ctx context.Context) (*organization.Organization, error) {
```

Comment:
This is a breaking change, and most likely will cause issues with the terraform client.

Please either look at keeping same function signature, or create a new function.

Reply:
Good point, I changed the fn signature back.